content/swiftui/concepts/notifications/notifications.md

```diff
@@ -0,0 +1,202 @@
+---
+Title: 'Notifications'
+Description: 'Learn how to use notifications in SwiftUI for event handling and communication.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'Notifications'
+Table of Contents:
+- Introduction
+- Alerts
+- Action Sheets
+- Toasts (Message Overlays)
+- Custom Notifications
+- Best Practices
+- Conclusion
+---
+# SwiftUI Notifications
+
+## Introduction 
+
+In **[SwiftUI](https://developer.apple.com/documentation/swiftui/)**, **`notifications`** refer to visual and interactive user interface components that display messages, alerts, or updates to the user. 
+
+They are used to provide information, prompt user interaction, or display important messages within your app's user interface. 
+
+SwiftUI provides several types of notifications that we can use to enhance the user experience. We can use modifiers and view modifiers to create and disply notifications as part of the interface. 
+
+If in Swift (using NotificationCenter) notifications are primarly a mechanism for communication between different app components, in SwiftUI notifications are visual elements used to present information or gather inputs from the user. 
+
+The two concepts are related and they both involve some form of communication or interaction, but they serve different purposes and are implemented differently inthe respective frameworks.
+
+
+Next, we are going to explore some of the SwiftUI Notifications types, scenarios where they are useful, and how to implement them.
```

Comment:
[Proofreading] It would be more beneficial to turn the paragraph into a heading title like the one below. That way, readers can see the different types and their functions as opposed to you announcing them. 
```suggestion
## Types of Swift UI Notficiations
```